MD/md-71/README.md

```diff
@@ -0,0 +1,39 @@
+# MD-71: Informer service for the Lock/Mint-based Native Bridge
+- **Description**: Provide a component that informs about the total circulating supply of \$MOVE across L1 and L2.
+- **Authors**: [Author](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Overview
+
+Several components should react if the bridge is under attack or faulty. In particular, the considered components are the Security Fund, see [MIP-50](https://github.com/movementlabsxyz/MIP/pull/50) and the Rate Limiter, see [MIP-56](https://github.com/movementlabsxyz/MIP/pull/56), and more generally the governance operator.
+
+The Operator that controls these components requires knowledge about the states of minted or locked tokens on L1 and L2. Moreover, the operation of these components may be handled via a governance, which could also rely on state information.
+
+This MD requests to provide an informing service that satisfies the above.
+
+## Desiderata
+
+### D1: Provide information about the circulating supply on L2
+
+On the L2 the bridge contract mints \$MOVE token when token should be transferred to L2. When tokens are transferred to L1, the user burns provably some tokens. The Informer should learn about those two supplies and provide the current circulating \$MOVE supply.
```

Comment:
Do we define the current circulating supply as only what is transferred over the bridge? Is there not a case where there is L1 MOVE that has not been transferred? Do we consider this part of the circulating MOVE supply?

Reply:
whether inflight tokens are part of the overall circulating supply is debatable, but you are correct this is the circutating supply on L2. i have clarified in text.